api-catalog-ui/frontend/package.json

```
@@ -63,6 +63,7 @@
         "test": "react-app-rewired test --runInBand --silent --watchAll=false --env=jsdom components/* utils/* reducers/* epics/* actions/* selectors/* ErrorBoundary/* helpers/* --reporters=default --reporters=jest-html-reporter --coverage",
         "cy:open": "cypress open",
         "cy:e2e:ci": "cypress run --spec \"cypress/e2e/**/*.cy.js\" --env catalogHomePage=https://gateway-service:10010/apicatalog/ui/v1,loginUrl=https://gateway-service:10010/apicatalog/api/v1/auth/login,gatewayOktaRedirect=https://gateway-service:10010/gateway/oauth2/authorization/okta?returnUrl=https%3A%2F%2Fgateway-service%3A10010%2Fapplication%2Fversion --browser chrome --headless",
+        "cy:e2e:ci:modulith": "cypress run --spec \"cypress/e2e/**/*.cy.js\" --env catalogHomePage=https://apiml:10010/apicatalog/ui/v1,loginUrl=https://apiml:10010/apicatalog/api/v1/auth/login,gatewayOktaRedirect=https://apiml:10010/gateway/oauth2/authorization/okta?returnUrl=https%3A%2F%2Fapiml%3A10010%2Fapplication%2Fversion --browser chrome --headless",
```

```suggestion
        "cy:e2e:ci:modulith": "cypress run --spec \"cypress/e2e/**/*.cy.js\" --env catalogHomePage=https://apiml:10010/apicatalog/ui/v1/index.html,loginUrl=https://apiml:10010/apicatalog/api/v1/auth/login,gatewayOktaRedirect=https://apiml:10010/gateway/oauth2/authorization/okta?returnUrl=https%3A%2F%2Fapiml%3A10010%2Fapplication%2Fversion --browser chrome --headless",
```

To avoid redirects in the next PR #4189

---

ok.. this is still wip though

---

This we will have to do again in the next PR, since this change is not yet in effect, if we make the change now it just fails